### PR TITLE
AddedManagingStudentProfile

### PR DESCRIPTION
**Pull request by KD54321**
Added css and react function for managing student profile
Added bootstrap script in index file.

### CODE REVIEW

**Review by Amir8Abdullah — COMMENTED** [review]
We want to use React for this project. Change HTML to react.

File: app/public/index.html [hunk 1]
```diff
@@ -24,7 +24,12 @@
       work correctly both with client-side routing and a non-root public URL.
       Learn how to configure a non-root public URL by running `npm run build`.
     -->
-    <title>React App</title>
```
Comment (Amir8Abdullah) [inline app/public/index.html:27]: Revert this change.

File: app/public/index.html [hunk 2]
```diff
@@ -24,7 +24,12 @@
...
+    <link
```
Comment (Amir8Abdullah) [inline app/public/index.html:27]: From line 27-31, what does this do?

File: app/src/App.tsx [hunk 3]
```diff
@@ -5,12 +5,14 @@ import './Header/Header.css';
...
 import LoginPage from "./LoginPage/LoginPage";
 import './LoginPage/LoginPage.css'
+import Manage_Profile from './ManagingStudentProfile/Manage_Profile';
 
 function App() {
   return (
     <>
       <Header />
       <LoginPage />
+      <Manage_Profile/>
```
Comment (Amir8Abdullah) [inline app/src/App.tsx:15]: You cannot have two pages in a same page. Remove this line.

**Review by Amir8Abdullah — CHANGES_REQUESTED** [review]
Needs changes.

File: app/public/index.html (hunk 2 above)
Comment (KD54321) [reply app/public/index.html:27]: its a bootstrap script that links to a css library

File: app/src/App.tsx (hunk 3 above)
Comment (KD54321) [reply app/src/App.tsx:15]: How are we going to implement this page then?

File: app/public/index.html (hunk 2 above)
Comment (Amir8Abdullah) [reply app/public/index.html:29]: Oh ok.

**Pull request merged**